Automattic/wp-calypso
# Stats: Updating purchase button

Related to #84740

## Proposed Changes

* pass purchase quantity to the checkout process

## Testing Instructions



* go to the live branch and visit the Stats purchase page
* navigate to a commercial licence and verify that the old flow is not affected by checking that you are redirected to a checkout page (don't purchase it)
* make sure that your store is running in a sandbox mode and you applied tiered billing patch
* go back to the Stats purchase page and apply the `stats/tier-upgrade-slider` feature flag 
* select a desired tier and purchase it
* switch your sandbox to trunk
* check the API response to `/sites/<id>/jetpack-stats/usage` and verify that `views_limit` corresponds to the one you purchased

## Pre-merge Checklist
